docs/1.Introduction/Getting Started.md

```diff
@@ -9,13 +9,28 @@ order: 2
 
 The installation is quick and straightforward - install, configure input & scan.
 
+## Prerequisites:
```

Comment:
Adding an article under "concepts" on variable rendering explaining how it is done and why would also be useful :)